src/content/1.7/testing/web-acceptance-tests/how-to-contribute-and-create-web-acceptance-tests.md

```diff
@@ -0,0 +1,212 @@
+---
+title: How to contribute and create web acceptance tests
+menuTitle: Contributing to the creation of web acceptance tests
+weight: 2
```

Comment:
You need to create an alias for the previous route you deleted, or else there will be broken links and lost page positioning

```suggestion
weight: 2
aliases:
   - /1.7/testing/how-to-create-your-own-web-acceptance-tests
```